### ISSUE TITLE
Test failure: p2p_sync_throttle_test

### ISSUE THREAD

**Issue by heifner** [AntelopeIO/spring#821]
https://github.com/AntelopeIO/spring/actions/runs/11021862404/job/30610476067

```
2024-09-24T21:20:30.228163      File "/__w/spring/spring/build/tests/p2p_sync_throttle_test.py", line 135, in <module>
    assert throttledNode.waitForBlock(endLargeBlocksHeadBlock, timeout=240), f'Wait for block {endLargeBlocksHeadBlock} on throttled node timed out'
```

**Comment by linh2931** [comment]
Just a note, this still happens in 1.0.1 as expected:
https://github.com/AntelopeIO/spring/actions/runs/11076494019/job/30780277759?pr=827#step:4:1839

**Comment by heifner** [comment]
Re-open issue to backport fix from `main` to `release/1.0`.
